### PR TITLE
Adding example for FastMarching Segmentation

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Comment by blowekamp** [comment]
TODO: 
- Add a Documentation.rst file
- Test the CSharp example ( no CI is testing CSharp )

**Review by zivy — COMMENTED** [review]
LGTM. Only comment I had already appeared in the TODO, add the Documentation.rst and update the index.rst with this new example.

**Comment by blowekamp** [comment]
@zivy @dave3d  Ready for review.

File: Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+using System;
+using System.Globalization;
+
+using itk.simple;
+using PixelId = itk.simple.PixelIDValueEnum;
+using SitkImage = itk.simple.Image;
+
+namespace itk.simple.examples
+{
+    public class Program
```
Comment (zivy) [inline Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs]: Possibly change the name of the class from "Program" to something meaningful?

**Review by zivy — COMMENTED** [review]
LGTM. Very well written documentation, more detailed than most of our examples, well done.
Only had one potential addition to the documentation and a comment with respect to the class name in C#.

File: Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs (hunk 1 above)
Comment (blowekamp) [reply Examples/FastMarchingSegmentation/FastMarchingSegmentation.cs]: done

**Comment by blowekamp** [comment]
@dave3d I'm unable to reproduce the C# compiler errors seen here: 
https://open.cdash.org/viewBuildError.php?buildid=5863587

Could you give it a try with VS2017?

**Comment by blowekamp** [comment]
Actually, I'm giving it on last try with using `Convert.ToDouble`.

**Pull request merged**